integration-tests/utils/Tokens.ts

```diff
@@ -0,0 +1,40 @@
+export class Tokens {
+
+    static removeBindingsAndTokens(){
+        cy.getCookie('cs_jwt').should('exist').then((cookie) => { 
+            let token = cookie.value;
+            let namespace = Cypress.env("USERNAME").toLowerCase();
+
+            removeAllResources(token, namespace, "spiaccesstokenbinding");
+            removeAllResources(token, namespace, "spiaccesstoken");
+        });
+    } 
+}
+
+function removeAllResources(token: string, namespace: string, resourceType : string) {
+    const getResources = {
+        method: 'GET',
+        url: `https://env-ephemeral-alqcwz-yjsfmfej.apps.c-rh-c-eph.8p0c.p1.openshiftapps.com/api/k8s/apis/appstudio.redhat.com/v1beta1/namespaces/${namespace}/${resourceType}s`,
```

Comment:
the only reason this url works is because there is a long lived ephemeral env behind it

this needs to be parameterized